ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts

```diff
@@ -424,6 +468,30 @@ test.describe('Attachment Point Tool', () => {
     expect(smiFile).toEqual(smiFileExpected);
   });
 
+  test.fixme('Click and Save as *.smi file', async ({ page }) => {
```

Comment:
Could you add a comment, why it is marked as `fixme`?

Reply:
This is a mistake. I will delete it with the next push of changes.

Reply:
I deleted it.